tests/services/test_room_collection_day_variant_merge.py

```
@@ -0,0 +1,212 @@
+        """3개 이상 variant → 병합 조건 미충족, 모두 원형 유지"""
+        rooms = [
+            _room("r-1", "[평일 낮] 1번룸", price=10000),
+            _room("r-2", "[주말] 1번룸", price=15000),
+            _room("r-3", "1번룸", price=12000),
+        ]
+        parsed = {
+            "r-1": _parsed("1번룸 (평일 낮)", day_type=None),
+            "r-2": _parsed("1번룸", day_type="weekend"),
+            "r-3": _parsed("1번룸", day_type=None),
+        }
+        merged_rooms, merged_parsed = service._merge_day_variant_rooms(rooms, parsed)
```

Ruff (0.15.6)

[warning] Unpacked variable `merged_parsed` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)